tests/models_tests/models_test.go

```diff
@@ -367,45 +366,45 @@ func TestRetrieveHistogramDataBySourceIdAndCohortIdAndConceptIdsAndCohortPairs(t
 	}
 }
 
-func TestQueryFilterByConceptIdsAndCohortPairsHelper(t *testing.T) {
-	// This test checks whether the query succeeds when there is a table or
-	// view aliased as "observation" and whether it fails otherwise.
-
-	setUp(t)
-	omopDataSource := tests.GetOmopDataSource()
-	filterConceptIds := []int64{1, 2, 3}
-	filterCohortPairs := []utils.CustomDichotomousVariableDef{} // empty / not really needed for test
-	var personIds []struct {
-		PersonId int64
-	}
-
-	// Subtest1: correct alias "observation":
-	query := omopDataSource.Db.Table(omopDataSource.Schema + ".observation_continuous as observation").
-		Select("observation.person_id")
-	query = models.QueryFilterByConceptIdsAndCohortPairsHelper(query, filterConceptIds, filterCohortPairs, omopDataSource.Schema, "")
-	meta_result := query.Scan(&personIds)
-	if meta_result.Error != nil {
-		t.Errorf("Did NOT expect an error")
-	}
-	// Subtest2: incorrect alias "observationWRONG"...should fail/panic:
-	defer func() {
-		panicMessage := recover()
-
-		if panicMessage == nil {
-			t.Errorf("The code did not panic")
-		}
-		panicMessageStr, isString := panicMessage.(string)
-
-		if !isString || !strings.HasPrefix(panicMessageStr, "Error: this QueryFilterByConceptIdsAndCohortPairsHelper is meant for ") {
-			t.Errorf("The code did not panic with expected error message")
-		}
-
-	}()
-	query = omopDataSource.Db.Table(omopDataSource.Schema + ".observation_continuous as observationWRONG").
-		Select("*")
-	query = models.QueryFilterByConceptIdsAndCohortPairsHelper(query, filterConceptIds, filterCohortPairs, omopDataSource.Schema, "")
-	query.Scan(&personIds)
-}
+// func TestQueryFilterByConceptIdsAndCohortPairsHelper(t *testing.T) {
```

Comment:
We will want to revisit this before merging to master though

Reply:
yes, makes sense. Let me follow-up on this.

Reply:
I changed the approach, so this issue is outdated now. Marking as solved.